public/resources/scss/shared.scss

```diff
@@ -2861,6 +2861,10 @@ inventory-view {
   padding-bottom: 30px;
 }
 
+.tier-completion {
+  float: right;
```

Comment:
I thought floats are evil? Or not?

~ https://github.com/SkyCryptWebsite/SkyCrypt/pull/1038#issuecomment-998763973

Reply:
Other than that it's great.

(might take a while until I figure out how to merge this as this is my first time doing that - hope I get this right...)

Reply:
> I thought floats were evil? Are they? ~ [SkyCryptWebsite#1038 (comment)](https://github.com/SkyCryptWebsite/SkyCrypt/pull/1038#issuecomment-998763973)

yeah the more I thought about it the more float makes sense in this case. I still don't like it but I didn't change it because what you have works fine